src/rtc/h5_prep.py

```diff
@@ -527,11 +545,11 @@ def get_metadata_dict(product_id: str,
              ' Reformatted, unprocessed instrument data; L1: Processed'
              ' instrument data in radar coordinates system; and L2:'
              ' Processed instrument data in geocoded coordinates system'],
-        'identification/productID':
-            ['product_id',
-             ALL_PRODUCTS,
-             product_id,
-             'Product identifier'],
+        # 'identification/productID':
+        #    ['product_id',
+        #     ALL_PRODUCTS,
+        #     product_id,
+        #     'Product identifier'],
```

Comment:
They are renaming the file names after the SAS is completed. The renaming would change the product ID because the product ID is basically the common prefix for the files. To avoid inconsistencies, I'm commenting it from the metadata.